keps/sig-node/2133-kubelet-credential-providers/README.md

```diff
@@ -376,46 +376,51 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
 
-TBD for beta.
+  Feature is enabled but exec plugin does not properly fetch and return credentials to the kubelet.
+  Impact is that kubelet cannot authenticate and pull credentials from those registries.
 
 * **What specific metrics should inform a rollback?**
 
-TBD for beta.
+  This feature does not have metrics.
```

Comment:
Metrics for how a credential exec plugin is performing seems pretty reasonable and low cardinality.  And it would be used to inform how well this feature and the exec plugin itself are working.

Reply:
That's fair, I'll add something about this